src/os/exec.go

```diff
@@ -47,6 +47,10 @@ func (p *ProcessState) Sys() interface{} {
 	return nil // TODO
 }
 
+func (p *ProcessState) Exited() bool {
+	return false // TODO
+}
```

Comment:
That's... a little ugly.
Is there any way we can track whether the process is still running?

Reply:
For now, I stubbed it out the same way all the other methods over `ProcessState` (which is an empty struct) are implemented. Making this functional would mean a quite large rework of the tinygo process representation, which atm is just a pid. I think it is a good idea to rework this, but not as part of this PR.